### DISCUSSION TITLE
Mistaken account blocking

### DISCUSSION THREAD

**Discussion by DimaD10** [ScratchAddons/ScratchAddons#7277] [Help] [answered]
I`ve been trying to load the gif (as my banner to readme,md profile) and I was thrown out from my GitHub account after. After this actions Ive got some letter to my email about password reset:

"Hello @DimaD10,

We recently noticed that your GitHub account had a suspicious login. Out of an abundance of caution, we made the decision to force a password reset for the account associated with this email address."

And then I can change the password and login to my account however my GitHub profile (and repositiry deploys) isnt visible for other users

PLS Help me, I cant work ;(

**Comment by wirierfungus76** [comment, marked as answer]
I don't know what else to say, but you can go to [this link](https://support.github.com/contact) to contact GitHub support and request help there. Have a good day!

**Reply by wirierfungus76** [reply]
yes I know this is outdated but I thought I should help anyways
